pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -90,16 +109,42 @@ stdenv.mkDerivation rec {
       "-DKICAD_STDLIB_DEBUG=ON"
       "-DKICAD_USE_VALGRIND=ON"
     ]
+    ++ optionals (sanitize) [
+      "-DKICAD_SANITIZE=ON"
+    ]
+    ++ optionals (withWayland) [
+      "-DKICAD_USE_EGL=ON"
+    ]
   ;
 
-  nativeBuildInputs = [ cmake doxygen pkg-config lndir ];
+  nativeBuildInputs = [
+    cmake
+    doxygen
+    graphviz
+    pkg-config
+    lndir
+  ]
+  # wanted by configuration on linux, doesn't seem to affect performance
```

Comment:
But closure size and build time?

Reply:
i'll check that out

Reply:
no effect on closure size, i'll note that in the comment (i seem to recall the hash of the binary (sha256sum) (binary not wrapper) changed)

Reply:
and still ~15m build time